e2e/lib/database-config.js

```
@@ -0,0 +1,93 @@
+        };
+    }
+}
+
+/**
+ * Get database configuration for specific test environment
+ * @param {string} env - Test environment (e.g., 'ci', 'local')
+ * @returns {Object} Knex database configuration
+ */
+function getDatabaseConfigForEnv(env) {
+    switch (env) {
+        case 'ci':
```

ESLint

[error] Expected indentation of 4 spaces but found 8.

(indent)